code/data_structures/src/list/singly_linked_list/operations/find/search_element_python.py

```diff
@@ -0,0 +1,73 @@
+class Node:
+    '''
+    A Node in a singly-linked list
+
+    Parameters
+    -------------------
+    data :
+        The data to be stored in the node
+    next:
+        The link to the next node in the singly-linked list
+    '''
+
+    def __init__(self, data=None, next=None):
+        self.data = data
+        self.next = next
+
+    def __repr__(self):
+        ''' Node representation as required'''
+        return (self.data)
+
+
+class SinglyLinkedList:
+    def __init__(self):
+        self.head = None
+
+    def append(self, data):
+        '''
+        Inserts New data at the ending of the Linked List
+        Takes O(n) time
```

Comment:
Is order guaranteed? we could append head to new node and assign head to new node.